core/src/avm1/activation.rs

```diff
@@ -2616,6 +2615,8 @@ impl<'a, 'gc, 'gc_context> Activation<'a, 'gc, 'gc_context> {
                     {
                         child.object()
                     } else {
+                        // TODO(moulins): avoid extra alloc?
```

Comment:
I don't think it's possible; at least not without support for substrings in `AvmString`.

Reply:
This will probably be possible when we actually introduce a custom String type (for UTF16), so I'll remove these comments for now.